plugins/score-card/src/api/ScoringDataJsonClient.ts

```diff
@@ -49,14 +49,18 @@ export class ScoringDataJsonClient implements ScoringDataApi {
 
   public async getScore(
     entity?: Entity,
-  ): Promise<SystemScoreExtended | undefined> {
+  ): Promise<EntityScoreExtended | undefined> {
     if (!entity) {
       return undefined;
     }
     const systemName = entity.metadata.name;
     const jsonDataUrl = this.getJsonDataUrl();
-    const urlWithData = `${jsonDataUrl}${systemName}.json`;
-    const result: SystemScore = await fetch(urlWithData).then(res => {
+    let urlWithData = `${jsonDataUrl}${systemName}.json`;
```

Comment:
I do personally try to avoid `let`, it's a matter of taste I guess but wouldn't be more readable 
```typescript
  const urlWithData = this.shouldUseEntityRef() ?
      `${jsonDataUrl}${systemName}.json` : 
      `${jsonDataUrl}${entity.metadata.namespace}/${entity.kind}/${entity.metadata.name}.json`.toLowerCase();
```
? But up to you..

Reply:
I'll follow you conventions, but it won't be relevant anymore after dropping the backward compatibility.